pkg/controller/hibernation/hibernation_controller.go

```diff
@@ -579,7 +579,7 @@ func (r *hibernationReconciler) checkClusterRunning(cd *hivev1.ClusterDeployment
 		if readyCondition.Reason == hivev1.ReadyReasonPausingForClusterOperatorsToSettle &&
 			time.Since(readyCondition.LastProbeTime.Time) < clusterOperatorSettlePause {
 			remainingPause := clusterOperatorSettlePause - time.Now().Sub(readyCondition.LastProbeTime.Time)
-			logger.Info("waiting an additional %s for ClusterOperators to settle", remainingPause)
+			logger.Infof("waiting an additional %s for ClusterOperators to settle", remainingPause)
```

Comment:
Latent. But if we're gonna fix it... we should convert to structured logging.
```suggestion
			logger.WithField("timeRemaining", remainingPause).Info("still waiting for ClusterOperators to settle")
```